ED/protocol.html

```diff
@@ -1126,7 +1139,14 @@ <h3 property="schema:name">WebID-TLS</h3>
 
 
           <section id="authorization" inlist="" rel="schema:hasPart" resource="#authorization">
+
             <h2 property="schema:name">Authorization</h2>
+
+            <p><span about="" id="server-authorization" rel="spec:requirement" resource="#server-authorization"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> conform to either the Web Access Control specification [<cite><a class="bibref" href="#bib-wac">WAC</a></cite>] or the Access Control Policy specification [<cite><a class="bibref" href="#bib-acp">ACP</a></cite>].</span></span></p>
```

Comment:
Just wondering if "either" is an exclusive OR; and whether we should define the behavior more specifically. Obviously, we'd run in all sorts of trouble if a server supports both and the documents are not synced (but conversely, if a server knows how to keep them in sync, then why not). For simplicity, I could make it an explicit exclusive OR.

Also, in particular, we should mention (or point to) how a client can discover which one the server supports.